packages/framework/client-logger/app-insights-logger/api-report/app-insights-logger.api.md

```diff
@@ -0,0 +1,19 @@
+## API Report File for "@fluidframework/app-insights-logger"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ITelemetryBaseEvent } from '@fluidframework/common-definitions';
+import { ITelemetryBaseLogger } from '@fluidframework/common-definitions';
+
+// @public (undocumented)
```

Comment:
Let's be sure to document the undocumented items here.

Reply:
Everything should be documented now